params.go

```diff
@@ -277,6 +287,16 @@ func (p *Params) SetDefaults(appLabel, buildVersion, releaseName, releaseAction
 		p.Autoscale.CPUPercentage = 80
 	}
 
+	if p.Autoscale.Safety.PromQuery == "" {
+		p.Autoscale.Safety.PromQuery = fmt.Sprintf("sum(rate(nginx_http_requests_total{app='%v'}[5m])) by (app)", p.App)
+	}
+	if p.Autoscale.Safety.Ratio == 0 {
+		p.Autoscale.Safety.Ratio = 1
+	}
+	if p.Autoscale.Safety.ScaleDownRatio == 0 {
+		p.Autoscale.Safety.ScaleDownRatio = 1
+	}
+
```

Comment:
I guess `Delta` can default to 0.

Reply:
It does if you don't set it; the property is a float which defaults to 0.0.